src/lib/OpenEXR/ImfContext.h

```diff
@@ -0,0 +1,78 @@
+//
+// SPDX-License-Identifier: BSD-3-Clause
+// Copyright (c) Contributors to the OpenEXR Project.
+//
+
+#ifndef INCLUDED_IMF_CONTEXT_H
+#define INCLUDED_IMF_CONTEXT_H
+
+#include "ImfContextInit.h"
+
+#include "ImfHeader.h"
+
+#include <memory>
+
+OPENEXR_IMF_INTERNAL_NAMESPACE_HEADER_ENTER
+
+/// @brief Context provides
```

Comment:
did you mean "provider"?

Reply:
no, it's just my auto-fill emacs macro for starting a new class and me not yet filling in the doxygen comments until I know what the final class looks like :)

Reply:
lol "the default provides, so say we all"